src/exportMap/captureDependency.js

```diff
@@ -0,0 +1,60 @@
+export function captureDependency(
```

Comment:
```suggestion
export default function captureDependency(
```

Reply:
Why would `captureDependency` be the default and not, say, `captureDependencyWithSpecifiers` that is exported from the same file?

Reply:
because the filename is "captureDependency". a default export is what a module _is_, a named export is something a module _has_, and most modules should _be_ something.